pages/fundamentals/acknowledgements.html

```diff
@@ -1,21 +1,19 @@
 ﻿
-<h1>Acknowledgments</h1>
+<h1>감사의 말</h1>
 
 <p>
-Firstly, thanks to the whole Go community.
-Without an active and responsive community, this book couldn't have been finished.
+먼저, 모든 Go 커뮤니티에 감사드립니다.
+적극적으로 응답해주는 커뮤니티가 없었다면, 이 책을 완성하지 못했을 것입니다.
 </p>
 
 <p>
-I'd like to give special thanks to Ian Lance Taylor.
-Ian kindly answered my countless boring questions
-on go-nuts group and the Go project issue tracker.
-Ian's answers helped me clear many of my ever confusions in using Go.
+Ian Lance Taylor 씨에게는 특별히 감사를 전하고 싶습니다.
```

Comment:
'에게'를 '께'로 바꾸고 뒤에 '는'은 딱히 필요 없어 보입니다.

그리고 사람 이름 뒤에 '씨'를 쓰기로 한 건 알지만 감사의 말에서는 '님'이 나아보이는데 다들 어떻게 생각하시나요

```suggestion
Ian Lance Taylor 님께 특별히 감사를 전하고 싶습니다.
```

Reply:
개인적으로는 그냥 씨로 해도 무관할 것 같네요 ㅎㅎ

Reply:
`께`는 보통 님과 같은 존칭 뒤에 붙이는 존대 표현 조사라 씨를 사용하다면 `에게` 정도로 정리해도 되지 않을까 싶네요~!

Reply:
여러 의견을 조합하여...
일단 `~씨`를 기본으로 하였습니다
`께`의 경우에는 지현씨의 의견을 생각하여 `~분들께`와 같이 앞에 높이는 경우에 한하여 수정하였습니다